awx/main/management/commands/run_dispatcher.py

```diff
@@ -135,6 +135,13 @@ def handle(self, *arg, **options):
                     Queue(q, Exchange(q), routing_key=q)
                     for q in (settings.AWX_CELERY_QUEUES_STATIC + [get_local_queuename()])
                 ]
+                queues.append(
+                    Queue(
+                        get_local_queuename(),
+                        Exchange(get_local_queuename()),
+                        routing_key='dispatcher-{}-control'.format(get_local_queuename())
```

Comment:
the dispatcher already worked with the status command... that works rather differently via

https://github.com/ansible/awx/blob/fc7cf7fb49e1cfe565f9238b9d020787ed64e7b8/awx/main/management/commands/run_dispatcher.py#L105-L107

And that's not how the callback receiver works post-PR. But they both listen to a check-in kind of queue.